jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/actions/ShowLogsAround.kt

```diff
@@ -24,9 +25,9 @@ class ShowLogsAroundGroup(
     }
 
     override fun getChildren(e: AnActionEvent?): Array<AnAction> = arrayOf(
-        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.one_minute"), 60 * 1000),
-        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.five_minutes"), 5 * 60 * 1000),
-        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.ten_minutes"), 10 * 60 * 1000)
+        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.one_minute"), Duration.ofMinutes(1).toMillis()),
+        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.five_minutes"), Duration.ofMinutes(5).toMillis()),
+        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.ten_minutes"), Duration.ofMinutes(10).toMillis())
```

Comment:
Why not pass `Duration` around and only convert to millis when we absolutely have to?

Reply:
Sure, plumbed Duration through